db/fixtures/ae_datastore/ManageIQ/Service/Provisioning/StateMachines/Methods.class/__methods__/catalogbundleinitialization.rb

```diff
@@ -38,14 +38,32 @@ def override_service_attribute(dialogs_options_hash, attr_name)
 
   log_and_update_message(:info, "Processing override_attribute for #{service_attr_name}...", true)
   attr_value = dialogs_options_hash.fetch(service_attr_name, nil)
-  attr_value = "#{@service.name}-#{Time.now.strftime('%Y%m%d-%H%M%S')}" if attr_name == 'name' && attr_value.nil?
+  attr_value = "#{@service.name}-#{Time.zone.now.strftime('%Y%m%d-%H%M%S')}" if attr_name == 'name' && attr_value.nil?
 
   log_and_update_message(:info, "Setting service attribute: #{attr_name} to: #{attr_value}")
   @service.send("#{attr_name}=", attr_value)
 
   log_and_update_message(:info, "Processing override_attribute for #{service_attr_name}...Complete", true)
 end
 
+def override_service_retirement_days(dialog_options_hash, _attr_name)
+  new_service_retirement_days = dialog_options_hash.fetch(:service_retirement_days, nil)
+  return if new_service_retirement_days.blank?
+
+  log_and_update_message(:info, "Processing override_service_retirement_days: #{new_service_retirement_days}", true)
+  @service.retires_on = Time.zone.today + new_service_retirement_days.to_i
```

Comment:
@billfitzgerald0120 
Could you add a log line here to match the log in catalogiteminitialization? 
log_and_update_message(:info, "Service retirement days: #{new_service_retirement_days}")